rotkehlchen/chain/polygon_pos/modules/hop/constants.py

```diff
@@ -0,0 +1,74 @@
+from typing import Final
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
+from rotkehlchen.chain.evm.decoding.hop.types import HopBridge
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.assets import A_WETH_POLYGON, A_WMATIC
+
+ETH_BRIDGE: Final = string_to_evm_address('0xb98454270065A31D71Bf635F6F7Ee6A518dFb849')
+ETH_AMM_WRAPPER: Final = string_to_evm_address('0xc315239cFb05F1E130E7E28E603CEa4C014c57f0')
+
+USDC_BRIDGE: Final = string_to_evm_address('0x25D8039bB044dC227f741a9e381CA4cEAE2E6aE8')
+USDC_AMM_WRAPPER: Final = string_to_evm_address('0x76b22b8C1079A44F1211D867D68b1eda76a635A7')
+
+USDT_BRIDGE: Final = string_to_evm_address('0x6c9a1ACF73bd85463A46B0AFc076FBdf602b690B')
+USDT_AMM_WRAPPER: Final = string_to_evm_address('0x8741Ba6225A6BF91f9D73531A98A89807857a2B3')
+
+MATIC_BRIDGE: Final = string_to_evm_address('0x553bC791D746767166fA3888432038193cEED5E2')
+MATIC_AMM_WRAPPER: Final = string_to_evm_address('0x884d1Aa15F9957E1aEAA86a82a72e49Bc2bfCbe3')
+
+DAI_BRIDGE: Final = string_to_evm_address('0xEcf268Be00308980B5b3fcd0975D47C4C8e1382a')
+DAI_AMM_WRAPPER: Final = string_to_evm_address('0x28529fec439cfF6d7D1D5917e956dEE62Cd3BE5c')
+
+HOP_BRIDGE: Final = string_to_evm_address('0x58c61AeE5eD3D748a1467085ED2650B697A66234')
+HOP_AMM_WRAPPER: Final = ZERO_ADDRESS
+
+A_USDC_POLYGON: Final = Asset('eip155:137/erc20:0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174')
+A_USDT_POLYGON: Final = Asset('eip155:137/erc20:0xc2132D05D31c914a87C6611C10748AEb04B58e8F')
+A_DAI_POLYGON: Final = Asset('eip155:137/erc20:0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063')
+A_HOP_POLYGON: Final = Asset('eip155:137/erc20:0xc5102fE9359FD9a28f877a67E36B0F050d81a3CC')
+A_HUSDC_POLYGON: Final = Asset('eip155:137/erc20:0x9ec9551d4A1a1593b0ee8124D98590CC71b3B09D')
+A_HUSDT_POLYGON: Final = Asset('eip155:137/erc20:0x9F93ACA246F457916E49Ec923B8ed099e313f763')
+A_HMATIC_POLYGON: Final = Asset('eip155:137/erc20:0x712F0cf37Bdb8299D0666727F73a5cAbA7c1c24c')
+A_HDAI_POLYGON: Final = Asset('eip155:137/erc20:0xb8901acB165ed027E32754E0FFe830802919727f')
+A_HETH_POLYGON: Final = Asset('eip155:137/erc20:0x1fDeAF938267ca43388eD1FdB879eaF91e920c7A')
+
+BRIDGES: Final = {
+    ETH_BRIDGE: HopBridge(
+        address=ETH_BRIDGE,
+        asset=A_WETH_POLYGON,
+        amm_wrapper=ETH_AMM_WRAPPER,
+        hop_asset=A_HETH_POLYGON,
```

Comment:
be consistent with the style. In the other places you have this argmuments in one line for all. Do like you do here for the other files since it follows our style